content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,177 @@
+save_as: tba
+url: tba
+date: 2020-04-02 23:40
+author: emus
+blog_id: newsletter
+category: Newsletter
+
+---
+
+# Collaboration over the Internet! XX May 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of April 2020.
+
+We are always happy about contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. The drafting process for each newsletter [is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+Furthermore, we started drafting each new version in the [XSF Github repository](https://github.com/xsf/xmpp.org/pulls) - feel free to add information by yourself!
+
+[Subscribe](https://tinyletter.com/xmpp) to receive the next edition in your inbox!
+
+
+#### Newsletter translations
+
+The translations of the XMPP Newsletter will be released here:
+
+- [German](https://www.jabber.de/)
+- [French](https://linuxfr.org/tags/xmpp/public)
+- [Spanish](https://www.jabber.de/)
+
+
+## XMPP Standards Foundation
+
+The XSF is renewing its sponsorship for 2020. Many thanks to everyone supporting and especially sponsoring in 2019. We would be happy if your organization were also considering [sponsoring also in 2020](https://mail.jabber.org/pipermail/standards/2020-April/037254.html)! If your organization considers sponsoring for the first time, [our mission may interest you as well](https://xmpp.org/about/xsf/mission).
+
+Alexander Gnauck published the [Membership Applications Q2 2020](https://wiki.xmpp.org/web/Membership_Applications_Q2_2020). Applications are welcome!
+
+
+## Articles
+
+[Pasis](https://github.com/pasis), maintainer of libstrophe and contributor to profanity, wrote about using the [XML console in profanity](https://profanity-im.github.io/blog/post/xml-console-in-profanity/) for debugging.
+
+Jan Cieśla from MongooseIM wrote a blog post about how [they thrive their project by the metrics](https://www.erlang-solutions.com/blog/how-data-drives-mongooseim.html).
+
+Can you believe? Zoom uses XMPP for its chat - cool! [Vulnerability](https://blog.talosintelligence.com/2020/04/zoom-user-enumeration.html) - not cool.
+
+MattJ (prosody developer) mentioned that, according to Debian popcon statistics, the prosody installations [have doubled since march](https://mastodon.technology/@mattj/104053716175271438).
+
+## Software releases
+
+[Kaidan 0.5.0](https://www.kaidan.im/2020/04/06/kaidan-0.5.0/) - Bam! Check out their new onboarding!
+
+Let's sum it up like this: Remove OTR + Rework conference and contact details + Show PDF previews + Add title for audio files = [Pix-Art Messenger 2.3.5](https://github.com/kriztan/Pix-Art-Messenger/blob/master/CHANGELOG.md)
+
+Four letters for Windows 10 Users (what ever that Windows 10 is): [UWPX 0.25.0](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.25.0.0) is online with Windows10X support and a push server. So give it a try!
```

Comment:
"whatever that Windows 10 is" => I do understand this in 2 ways:
 1/ I don't know what is WIndows 10, I don't want to know, and I'm good with my ignorance
 2/ whatever the version of Windows 10 is used

If 2/ then it should be worded more accurately
if 1/ then I find it unnecessarily dismissive: I don't think this is the place to bash such and such OS

Reply:
Actually, my intention was not to bash windows, but make fun of that many here just don't develop with it and pretend it is not present at all. Will remove it